modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -2485,13 +2556,22 @@ private void releaseCell(T cell) {
         }
     }
 
+
     /**
-     * This method is an experts-only method - if the requested index is not
-     * already an existing visible cell, it will create a cell for the
-     * given index and insert it into the sheet. From that point on it will be
-     * unmanaged, and is up to the caller of this method to manage it.
+     * Creates and returns a new cell for the given index.
+     * <p>
+     * If the requested index is not already an existing visible cell, it will create a cell for the given index and
+     * insert it into the sheet. If the index is existing, simply returns the visible cell. From that point on it will
+     * be unmanaged, and is up to the caller of this method to manage it.
+     * <p>
```

Comment:
* "sheet" is the table? if so, use that instead of a new term, otherwise explain what "sheet" is.
* "index is existing" -> "index exists"
* "From that point on" comma after "on"